objects/9ad187/ui.xml

```diff
@@ -10,7 +10,7 @@
     <Row class="spacing" preferredHeight="30"/>
     <Toggle textColor="#FFFFFF"/>
     <Slider wholeNumbers="true"/>
-    <Toggle scale="2 2 1"/>
+    <Toggle scale="2 2 1" fontSize="44"/>
```

Comment:
i dont think this change is needed? the reason why i didn't use toggle's actual text is that there's no way to scale the checkbox independent of the text that i could find. which lead to some really awkward artifacts

Reply:
Yeah, I was playing around trying to achieve something similar.